### PR TITLE
bumping the required go version to the latest homebrew version

### PR DESCRIPTION
**Pull request by cblkwell**

### CODE REVIEW

File: bin/check_go_version [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 set -eu -o pipefail
 
-VERSION="go1.11"
+VERSION="go1.12.2"
```
Comment (chrisgilmerproj) [inline bin/check_go_version]: How about just `1.12`? and leave the patches alone.

Comment (cblkwell) [reply bin/check_go_version]: Fixed! But it looks like CircleCI is unhappy. :(

**Comment by dynamike** [comment]
@cblkwell I think you'll need to bump the latest version of circleci-docker-primary because the Go version CircleCI is using is still 1.11. 

https://github.com/trussworks/truss-aws-tools/blob/master/.circleci/config.yml#L5

**Comment by cblkwell** [comment]
> @cblkwell I think you'll need to bump the latest version of circleci-docker-primary because the Go version CircleCI is using is still 1.11.
> 
> https://github.com/trussworks/truss-aws-tools/blob/master/.circleci/config.yml#L5

Yup! Thanks, looks like it's fixed now!

**Pull request merged**